libs/core/src/lib/avatar-group/avatar-group.component.scss

```diff
@@ -767,3 +767,14 @@
     color: #fff;
     color: var(--fdAvatar_Accent_Color,var(--sapContent_ContrastTextColor,#fff))
 }
+
```

Comment:
I think it won't a problem if you bring this in styles, merge and then use that rc in your PR

Reply:
Waiting for https://github.com/SAP/fundamental-styles/pull/2261

Reply:
approved

Reply:
@droshev updated